source/manual/deployments.html.md

```diff
@@ -0,0 +1,56 @@
+---
+owner_slack: "#govuk-developers"
+title: Deployments
+section: Deployment
+layout: manual_layout
+parent: "/manual.html"
+---
+
+On GOV.UK, all applications are set up for continuous deployment to integration, with the majority also having continuous deployment configured for staging and production.  Additionally, you can manually deploy a git reference (tag, branch, or commit SHA) of an application to any environment.
+
+Continuous integration (CI) is run on new commits on the main branch, usually from merging a pull request. If the commits pass CI, they are automatically deployed to integration. After deployment, a set of smoke tests run in integration to verify that the application still works as expected.
+
+If the smoke tests pass and the application has continuous deployment enabled, the deployment is promoted to staging. In staging another set of smoke tests run, and if they pass, the deployment will be promoted to production.
+
+For applications without continuous deployment enabled, they will need to be manually deployed to staging and production.
+
+Refer to the Release app to view current deployment information and identify which applications require manual deployment to later environments.
+
+## Manual deployments
+
+You can manually deploy an application by triggering the "Deploy" GitHub Action workflow for that application’s repository.
+
+This can be done in GitHub’s web interface:
+
+1. Go to the "Actions" page in the application repository
+1. Select the "Deploy" workflow from the list of workflows on the left hand side
+1. Click "Run workflow"
+1. Input the git reference and environment (ignore the "Use workflow from" option)
+1. Click "Run workflow"
+
+Or using GitHub’s CLI:
+
+```bash
+gh workflow run -R "alphagov/${REPO}" deploy.yml -F environment=${ENVIRONMENT} -F gitRef=${GIT_REF}
+```
+
+## Overview of the deployment process
+
+This is an example deploying an application to integration:
+
+1. Merge PR into main branch and successfully passes CI
+1. Triggers "Deploy" workflow in GitHub Action
+    1. Builds and pushes image to AWS Elastic Container Registery (ECR) in production
+    1. Sends webhook to Argo Workflows in production
+1. Triggers "deploy-image" workflow in Argo Workflows in production
+    1. Updates the image tag reference in govuk-helm-charts with a new commit
+    1. Adds a "deployed-to-<environment>" tag on the image in ECR
+    1. Notifies the Release application of the deployment
+1. Argo CD polls govuk-helm-charts and detects update image tag reference
+    1. Triggers sync of the application in Argo CD
```

Comment:
```suggestion
    1. Triggers sync of the `app-config` ([app-of-apps](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/#app-of-apps-pattern)) application in Argo CD, which updates the Helm values for the deployed app's Argo CD application resource
    1. The change to the deployed app's application resource triggers a sync of the deployed app
```

Reply:
dunno if this is too much detail but thought it might be helpful to mention the extra step that's involved here - feel free to use/edit/ignore!

Reply:
Yeah, was trying to quickly jot down the important steps - but I think this good to point out too!